packages/shared/lib/core/wallet/interfaces/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './asset-balance.interface'
+export * from './asset.interface'
```

Comment:
I think alphabetical order would have `asset.interface` before `asset-balance.interface`.

Reply:
Not according to list of files next in the ide, the - must be earlier